cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -0,0 +1,226 @@
+/* topdoc
+  name: Global Search molecule
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+          <div class="m-global-search">
+              <p class="short-desc">
+              </p>
+              <a class="btn btn__full" href="/">
+              </a>
+          </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-call-to-action
+            .short-desc
+            .btn.btn__full
+  tags:
+    - cfgov-molecules
+*/
+
+.m-global-search {
+
+    // Absolute position when there is a flyout at mobile/tablet sizes.
+    .respond-to-max( @bp-sm-max, {
+        // Make the height large so that the overflowing flyout can be hidden,
+        // but not get cropped when it's visible.
+        height: 500px;
+        width: 100%;
+        position: absolute;
+        top: 0;
+        overflow-x: hidden;
+        // TODO: This will be ignored in IE8-10,
+        //       leading to the overflow preventing clicks on the links.
+        pointer-events: none;
+    } );
+
+    // TODO: Move these positioning styles to the header organism.
+    .respond-to-min( @bp-med-min, {
+        float: right;
+        &_content {
+            padding-right: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
+        }
+    } );
+
+    &_trigger {
+        height: unit(60px / 24px, em);
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/62c72a886ebebaf94336e95873387f7b90c77a8a